src/main.js

```diff
@@ -71,3 +79,22 @@ function homeVisibility() {
   randomButton.classList.remove("hidden");
   saveCoverButton.classList.remove("hidden");
 }
+
+function createNewBook(event) {
+  event.preventDefault();
+  homeVisibility();
+  mainCoverSection.classList.remove("hidden");
+  var currentURL = window.location.href;
+  var coverValue = coverField.value;
+  var titleValue = titleField.value;
+  var desc1Value = desc1Field.value;
+  var desc2Value = desc2Field.value;
+  covers.push(currentURL);
```

Comment:
unnecessary ln do coverValue

Reply:
Got it! Also noticed that this makes line 87 unnecessary as well, I'll get that fixed!